ui/app/utils/default_jobs/variables.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable */
+export default `# Use Nomad Variables to modify this job's output:
+# run "nomad var put nomad/jobs/variables-example name=YOUR_NAME" to get started
+
+job "variables-example" {
+  datacenters =  ["dc1"]
+
+  group "web" {
+
+    network {
+      # Task group will have an isolated network namespace with
+      # an interface that is bridged with the host
+      mode = "bridge"
```

Comment:
This relies on CNI plugins being installed - is there a good way to do this without bridge?